scripts/Tools/check_case

```diff
@@ -1,65 +1,30 @@
-#!/usr/bin/env perl
+#!/usr/bin/env python2
 
-use strict;
-use Cwd;
-use English;
-use Getopt::Long;
-use File::Basename;
-use IO::File;
-use IO::Handle;
-use XML::LibXML;
-sub usage {
-    die <<'EOF';
+"""
+Checks that case is set up correctly
+"""
 
-SYNOPSIS
+from standard_script_setup import *
 
-  check_case [options]
+from CIME.utils import expect, get_model, run_cmd
+from CIME.case  import Case
 
-  checks that case is set up correctly
+logger = logging.getLogger(__name__)
 
-OPTIONS
+parser = argparse.ArgumentParser()
+CIME.utils.setup_standard_logging_options(parser)
 
-  -help [or -h]  Print usage to STDOUT (optional).
+caseroot = os.getcwd()
```

Comment:
Strongly agree. I think most of the performance issues with the perl scripts was the tendency to use shell commands to run things which then typically required a ton of repeated work like reparsing all the XML.